community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/MatchAcceptanceTest.scala

```diff
@@ -338,7 +337,7 @@ class MatchAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsTe
 
   test("finds a single path for paths of length one") {
     /*
-       a-b-c
+       (a)-(b)-c
```

Comment:
This comment is inconsistent (and some others, too). Not that it matters much I suppose.